### DISCUSSION TITLE
fancy_listing from tipboard 1.x

### DISCUSSION THREAD

**Discussion by MarcViola** [the-maux/Chartboard#167] [General]
Can someone port the fancy_listing tile template from tipboard 1.x ?
It would be appreciated.

Thanks.

**Comment by MarcViola** [comment]
> 

Look at me answering my own question again.  It is possible to "fancy" the regular listing template by encoding html for the text.

`curl -X POST http://<IP:port>/api/push -d tile_id=tile1 -d tile_template=listing -d 'data={"items":["<h1><span style=\"background-color: #00ff00;\">test1</span></h1>", "<h1><span style=\"background-color: #ff0000;\">test2</span></h1>"]}'`

**Reply by the-maux** [reply]
Hello 😃 ;

Look at me answering to a discussionissue that i don't even care (did you notice there is no "Issue" section ?, I don't think so).

But to answer you, yes it's possible to do that @MarcViola
I will be happy to review your PR and validate it 😄.

Look at the code, it's easy.

**Comment by the-maux** [comment]
If you did it, make PR ?
I will take a look
